py/selenium/webdriver/edge/webdriver.py

```diff
@@ -25,12 +26,16 @@
 class WebDriver(RemoteWebDriver):
 
     def __init__(self, executable_path='MicrosoftWebDriver.exe',
-                 capabilities=None, port=0, verbose=False, log_path=None):
+                 capabilities=None, port=0, verbose=False, service_log_path=None, log_path=None):
+        if log_path:
+            warnings.warn('use service_log_path instead of log_path', DeprecationWarning)
```

Comment:
Adding a `stacklevel` here and in the similar instances below, would help make the resultant warnings clearer, since they would reference the offending caller rather than the `.warn()` line in these files. I've filed #6274 for this (but it would be good to watch out for this when reviewing future PRs).